src/components/Actions/Actions.vue

```diff
@@ -497,6 +550,14 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+
+		/**
+		 * Display x items inline out of the dropdown menu
+		 */
+		inline: {
+			type: Number,
+			default: 0,
```

Comment:
```suggestion
			default: 1,
```
Otherwise it's a breaking change.
But I am not completely sold, as it conflicts a bit with the `forceMenu` prop.

Reply:
See the failing `Acitons` test.

Reply:
No, that seems counter intuitive. Let me fix the usage instead :)

> But I am not completely sold, as it conflicts a bit with the forceMenu prop.

Hum, Why though? You can force the menu if you want and if there is enough items to render there?

Reply:
I would say forceMenu should force all actions into the menu, also the inlined ones. But that's a matter of interpretation, I guess. And since it will keep it's behaviour unless you add inline=xx I am fine with both.

PS: I accidentally edited your above comment, instead of quoting it, and then made it worse by trying to restore it. Sorry for this 🙈

Reply:
That's a good question!
I can make it override any inline, sure! :)
I can even show a warning! let's do this!

> PS: I accidentally edited your above comment, instead of quoting it, and then made it worse by trying to restore it. Sorry for this :see_no_evil:

Ahah, no worries! :D

Reply:
Done, with testing added